hudi-utilities/src/main/java/org/apache/hudi/utilities/config/CloudSourceConfig.java

```diff
@@ -87,6 +87,13 @@ public class CloudSourceConfig extends HoodieConfig {
       .markAdvanced()
       .withDocumentation("Only selects objects in the bucket whose relative path starts with this prefix");
 
+  public static final ConfigProperty<String> SELECT_RELATIVE_PATH_REGEX = ConfigProperty
+      .key(STREAMER_CONFIG_PREFIX + "source.cloud.data.select.path.regex")
+      .noDefaultValue()
+      .withAlternatives(DELTA_STREAMER_CONFIG_PREFIX + "source.cloud.data.select.path.regex")
+      .markAdvanced()
+      .withDocumentation("Only selects objects in the bucket whose relative path matches this regex");
```

Comment:
In the docs, add an example of how it works with and without `source.cloud.data.select.relpath.prefix` based on the logic.

Reply:
Updated documentation